### PR TITLE
Update what-is-your-goal.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
add deep links to SQL, Python, and Web topics

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md?ref=f939327aefb6c21165be24b6aba9283b550a1d63)

File: coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md [hunk 1]
```diff
@@ -37,4 +37,4 @@ A querying language is a specialized programming language that is used to commun
 It can be used to retrieve information and alter the contents of the database. 
 
 [2:Data Analysis]
-If you're interested in analyzing data you should check out our Data Analysis topic.
\ No newline at end of file
+If you're interested in analyzing data you should check out our Data Analysis topic.
```
Comment (kapnobatai137) [inline coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md:40]: let's link to the data analysis topic as well
```suggestion
If you're interested in analyzing data you should check out our [Data Analysis](https://app.enki.com/skill/data-analysis) topic.
```

File: coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ links:
 At the end of this course, you'll have several options for the next step to take.
 
 Here are some suggestions from Enki:
-- Python is a popular language for beginners. It's easy to read, has a large community, and can be used for almost anything.
-- SQL is a *querying* language[1] and a good option as a stepping stone to learning programming languages and analyzing data[2].
-- If you want to start by making a website, make sure you go through our Web course (which includes HTML and CSS).
+- [Python](https://app.enki.com/skill/python) is a popular language for beginners. It's easy to read, has a large community, and can be used for almost anything.
+- [SQL](https://app.enki.com/skill/sql) is a *querying* language[1] and a good option as a stepping stone to learning programming languages and analyzing data[2].
+- If you want to start by making a website, make sure you go through our [Web](https://app.enki.com/skill/web) course (which includes HTML and CSS).
```
Comment (kapnobatai137) [inline coding-intro/coding-intro-core/coding-intro-introduction/what-is-your-goal.md:25]: Web is the topic; HTML and CSS are the courses
```suggestion
- If you want to start by making a website, make sure you go through our [Web](https://app.enki.com/skill/web) topic (which includes HTML and CSS).
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by nem035 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**